style.css

```diff
@@ -10,402 +10,664 @@ body {
+  line-height: 1.5rem;
+  margin: 0.75rem 1.5rem;
+}
+
+.myself .social {
+  margin: 0 0 1.3rem 1.5rem;
+}
+
+.myself .buttonlrg {
+  margin-left: 1.5rem;
+}
+
+.flexdwn {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 2px solid #dfe1e6;
+  margin: 0 6.4%;
+  padding: 5%;
+  font-family: 'Poppins', sans-serif;
+  color: #000;
+  font-weight: 500;
+  font-size: 20px;
+}
+
+.js ul {
+  display: grid;
+  align-content: center;
+  place-content: center;
+  grid-template-columns: repeat(1, 1fr);
+  justify-content: flex-start;
+  gap: 0.75rem;
+}
+
+.js li {
+  display: flex;
+  gap: 1rem;
+  margin: 0 9.6%;
+  background-color: #f7f7f9;
+  align-items: center;
+  list-style-type: none;
+  border-radius: 0.5rem;
+  padding: 2.4% 5%;
+  font-family: 'Poppins', sans-serif;
+  color: #253858;
+  font-weight: 500;
+  font-size: 0.93rem;
+}
+
+/* -----------------------contact me-------------------------- */
+
+.contact-me {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 6.5rem 2.4rem 0.5rem 2.4rem;
+  position: relative;
+  z-index: 1;
+}
+
+.bg {
+  margin-top: 11rem;
+  position: relative;
+  overflow: hidden;
+  background-color: #6070ff;
+  border-radius: 6rem 0 0 0;
+}
+
+.bg::after {
+  content: "";
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  width: 66%;
+  background-image: url('img/contactForm.png');
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  z-index: 0;
+}
+
+.contact-me h2 {
+  flex-basis: 100%;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.25rem;
+  color: #fff;
+  margin-bottom: 0.75rem;
+  text-align: center;
+}
+
+.contact-me p {
+  color: #ebebff;
+  font-size: 1.25rem;
+  font-weight: 400;
+  line-height: 1.75rem;
+  flex-basis: 100%;
+  text-align: center;
+  margin-bottom: 3.25rem;
+}
+
+.contact-me button {
+  width: 8.875rem;
+  height: 3rem;
+  border-radius: 0.5rem;
+  color: #6070ff;
+  background-color: #fff;
+  border: none;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 1.1rem;
+  line-height: 1.5rem;
+  transition: background-color 0.3s ease;
+}
+
+.butn:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.butn:active {
+  background-color: #2230d2;
+  color: #fff;
+  border-color: #2230d2;
+}
+
+.butn:disabled {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
+
+.contact-me form {
+  flex-basis: 100%;
+  display: flex;
+  flex-direction: column;
+  row-gap: 1.25rem;
+  width: 20.43rem;
+}
+
+.contact-me form input {
+  padding: 0.935rem 2.37rem 0.935rem 1rem;
+  border-radius: 0.5rem;
+  border: 0;
+}
+
+.contact-me form textarea {
+  padding: 0.9375rem 2.375rem 0.9375rem 1rem;
+  width: 20.4375rem;
+  height: 10.625rem;
+  border-radius: 0.5rem;
+  border: 0;
+}
+
+/* --------------------desktop version------------- */
+@media (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
```

Comment:
Your design is really impressive welldone✨. However, I noticed that you are using fixed widths for your elements which is causing your design to only look good on screen widths of 375px and 1440px. Please note that you are supposed to use responsive values for your element sizes so that the webpage scales properly whenever the screen size is increased or reduced. Take a look at the video below for reference:


https://github.com/Porkalaikarthika/Portfolio/assets/37116322/e90dca5b-bc76-47d9-9cfa-892b221aaf04



Please update your webpage to use responsive widths so that the webpage scales properly regardless of the screen size. Try using percentage widths (%) instead of rem🙏